stdlib/map.ml

```diff
@@ -497,6 +498,42 @@ module Make(Ord: OrderedType) = struct
 
     let choose_opt = min_binding_opt
 
+    let of_sorted_list l =
```

Comment:
If I understand correctly, this function is currently *not* exposed in `map.mli` and to users. You mentioned that you would like to make it available, but there seems to be rather a (weak) consensus *against* this idea, so it's good that it is currenty not exposed.